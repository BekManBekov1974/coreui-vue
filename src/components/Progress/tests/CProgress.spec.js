import { mount } from '@vue/test-utils'
import Component from "../CProgress";

const ComponentName = 'CProgress'
const wrapper = mount(Component,{
  propsData: {
    variant: 'black',
    value: 50
  }
})
// /* eslint-disable no-console */
// console.log(wrapper.vm.computedVariant)
// console.log(wrapper.vm.progressBarStyles)

describe(`${ComponentName} .vue`, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
  it('sets background color correctly', () => {

    expect(wrapper.element).toMatchSnapshot()
  })
});
