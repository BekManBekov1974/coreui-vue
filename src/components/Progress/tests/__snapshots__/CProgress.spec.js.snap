// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CProgress .vue renders correctly 1`] = `
<div
  class="progress"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="50"
    class="progress-bar"
    role="progressbar"
    style="width: 50%; background-color: black;"
  >
    <!---->
  </div>
</div>
`;

exports[`CProgress .vue sets background color correctly 1`] = `
<div
  class="progress"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="50"
    class="progress-bar"
    role="progressbar"
    style="width: 50%; background-color: black;"
  >
    <!---->
  </div>
</div>
`;
